File: backend/myalkansya/src/main/java/edu/cit/myalkansya/dto/AuthResponse.java
package edu.cit.myalkansya.dto;

public class AuthResponse {
    private String token;
    private GoogleUserDTO user;

    public AuthResponse(GoogleUserDTO user, String token) {
        this.user = user;
        this.token = token;
    }

    public String getToken() {
        return token;
    }

    public GoogleUserDTO getUser() {
        return user;
    }
}

File: frontend_mobile/myAlkansyaMob/app/src/main/java/com/example/myalkansyamobile/model/AuthResponse.kt
package com.example.myalkansyamobile.model

import com.google.gson.annotations.SerializedName

data class AuthResponse(
    @SerializedName("token")
    val token: String,

    @SerializedName("user")
    val user: UserDTO? // Changed to nullable and using UserDTO instead of GoogleUserDTO
)


